server/cache.ts

```diff
@@ -0,0 +1,21 @@
+import LRUCache from "lru-cache";
+
+const options = {
```

Comment:
probably should specify maxSize as well so that we can have a memory upper bound

Since we generate very similar images, so we should have a good guess of each image size and should comment it here. Thus we can have a rough estimate about the total mem usage here.